Library/Homebrew/utils/curl.rb

```diff
@@ -118,36 +118,37 @@ def curl(*args, print_stdout: true, **options)
       result
     end
 
-    def curl_download(*args, to: nil, partial: true, **options)
+    def parse_headers(headers)
+      return {} unless headers
```

Comment:
```suggestion
      return {} unless headers.present?
```

Reply:
Why `present?`?

Reply:
`if headers.blank?` is even nicer 😁

Reply:
Using `present?` or `blank?` (they're opposites) checks not only for `nil` or falsey but also for empty strings, whitespace-only strings, etc. We usually prefer them because they can catch a few more edge cases.

Quick overview https://blog.appsignal.com/2018/09/11/differences-between-nil-empty-blank-and-present.html

Reply:
Appreciate the link. Thanks :)